app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -13,7 +13,7 @@ def hawkular_entrypoint
     URI::HTTPS.build(
       :host => hawkular_endpoint_empty ? @ext_management_system.hostname : hawkular_endpoint.hostname,
       :port => hawkular_endpoint_empty ? worker_class.worker_settings[:metrics_port] : hawkular_endpoint.port,
-      :path => worker_class.worker_settings[:metrics_path])
+      :path => worker_class.worker_settings[:metrics_path] || '/hawkular/metrics')
```

Comment:
In the testing env the :metric_path is not declared :-( we need this for the tests to pass, 
It's a good fall-back for regular use as well.